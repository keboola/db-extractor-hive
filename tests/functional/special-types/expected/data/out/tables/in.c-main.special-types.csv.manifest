{"destination":"in.c-main.special-types","incremental":false,"metadata":[{"key":"KBC.name","value":"special_types"},{"key":"KBC.schema","value":"default"}],"column_metadata":{"id":[{"key":"KBC.datatype.type","value":"INTEGER"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"INTEGER"},{"key":"KBC.sourceName","value":"id"},{"key":"KBC.sanitizedName","value":"id"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false}],"bin":[{"key":"KBC.datatype.type","value":"STRING"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.sourceName","value":"bin"},{"key":"KBC.sanitizedName","value":"bin"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false}],"map":[{"key":"KBC.datatype.type","value":"STRING"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.sourceName","value":"map"},{"key":"KBC.sanitizedName","value":"map"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false}],"array":[{"key":"KBC.datatype.type","value":"STRING"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.sourceName","value":"array"},{"key":"KBC.sanitizedName","value":"array"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false}],"union":[{"key":"KBC.datatype.type","value":"STRING"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.sourceName","value":"union"},{"key":"KBC.sanitizedName","value":"union"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false}],"struct":[{"key":"KBC.datatype.type","value":"STRING"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.sourceName","value":"struct"},{"key":"KBC.sanitizedName","value":"struct"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false}]},"columns":["id","bin","map","array","union","struct"]}
