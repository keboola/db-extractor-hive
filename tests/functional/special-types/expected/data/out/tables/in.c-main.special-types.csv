"1","ASq8","{1:""item1"",2:""item2"",3:""item3""}","[""str1"",""str2""]","{0:123}","{""age"":20,""name"":""Name1""}"
"2","A03v","{4:""item4"",5:""item5"",6:""item6""}","[""str3"",""str4""]","{1:""456""}","{""age"":30,""name"":""Name2""}"
